drivers/pinctrl/pinctrl_rpi_pico.c

```diff
@@ -20,6 +20,7 @@ static void pinctrl_configure_pin(const pinctrl_soc_pin_t *pin)
 	gpio_set_input_hysteresis_enabled(pin->pin_num, pin->schmitt_enable);
 	gpio_set_input_enabled(pin->pin_num, pin->input_enable);
 	gpio_set_oeover(pin->pin_num, pin->oe_override);
+	gpio_set_outover(pin->pin_num, pin->out_override);
```

Comment:
While we're at it, we might as well add support for irqover and inover.

Reply:
Sure I have added both `irqover` and `inover`. I also run `clang-format` on the file as it was failing the `check_compliance.py` script locally.